src/main/java/com/ericsson/ei/handlers/RmqHandler.java

```diff
@@ -148,45 +153,55 @@ public ConnectionFactory connectionFactory() {
     }
 
     @Bean
-    Queue queue() {
+    public Queue externalQueue() {
         return new Queue(getQueueName(), true);
     }
 
     @Bean
-    TopicExchange exchange() {
+    public Queue internalQueue() {
+        return new Queue(getWaitlistQueueName(), true);
+    }
+
+    @Bean
+    public TopicExchange exchange() {
         return new TopicExchange(exchangeName);
     }
 
     @Bean
-    Binding binding(Queue queue, TopicExchange exchange) {
-        return BindingBuilder.bind(queue).to(exchange).with(bindingKey);
+    Binding binding() {
+        return BindingBuilder.bind(internalQueue()).to(exchange()).with(WAITLIST_BINDING_KEY);
     }
 
     @Bean
-    public SimpleMessageListenerContainer bindToQueueForRecentEvents(ConnectionFactory springConnectionFactory,
+    public List<Binding> bindings() {
+        String[] bingingKeysArray = splitBindingKeys(bindingKeys);
```

Comment:
You can also implement it as EI instance list in Frontend and read that list/array values directly to a ArrayList/Json object without String manipulation as you do here.

Reply:
And for backward compatibility, check if property value start with a '[', if not the just read as one bindingKey string.

Reply:
In that list and in the ldap server list they are "objects" with multiple key-values. In this case I would prefer to make it more simple and keep it as a string.